Parachain-staking benchmarks code related questions

Hi moonbeam team! 
I'm having some problems executing parachain-staking benchmarks code, hope you can help.

1.  when executing `cargo test -p pallet-parachain-staking --features runtime-benchmarks` command ,the terminal returned some errors
 ```
   Compiling pallet-parachain-staking v3.0.0 (/mnt/d/RustCode/moonbeam/pallets/parachain-staking)
error[E0599]: no function or associated item named `test_benchmark_hotfix_remove_delegation_requests` found for struct `pallet::Pallet` in the current scope
    --> pallets/parachain-staking/src/benchmarks.rs:1297:31
     |
1297 |             assert_ok!(Pallet::<Test>::test_benchmark_hotfix_remove_delegation_requests());
     |                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
     |                                        |
     |                                        function or associated item not found in `pallet::Pallet<Test>`
     |                                        help: there is an associated function with a similar name: `execute_delegation_request`
     |
    ::: pallets/parachain-staking/src/lib.rs:100:5
     |
100  |     pub struct Pallet<T>(PhantomData<T>);
     |     -------------------- function or associated item `test_benchmark_hotfix_remove_delegation_requests` not found for this struct

error[E0599]: no function or associated item named `test_benchmark_hotfix_update_candidate_pool_value` found for struct `pallet::Pallet` in the current scope
    --> pallets/parachain-staking/src/benchmarks.rs:1304:31
     |
1304 |             assert_ok!(Pallet::<Test>::test_benchmark_hotfix_update_candidate_pool_value());
     |                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
     |                                        |
     |                                        function or associated item not found in `pallet::Pallet<Test>`
     |                                        help: there is an associated function with a similar name: `test_benchmark_join_candidates`
     |
    ::: pallets/parachain-staking/src/lib.rs:100:5
     |
100  |     pub struct Pallet<T>(PhantomData<T>);
     |     -------------------- function or associated item `test_benchmark_hotfix_update_candidate_pool_value` not found for this struct

error[E0599]: no function or associated item named `test_benchmark_round_transition_on_initialize` found for struct `pallet::Pallet` in the current scope
    --> pallets/parachain-staking/src/benchmarks.rs:1507:31
     |
1507 |             assert_ok!(Pallet::<Test>::test_benchmark_round_transition_on_initialize());
     |                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
     |                                        |
     |                                        function or associated item not found in `pallet::Pallet<Test>`
     |                                        help: there is an associated function with a similar name: `test_benchmark_base_on_initialize`
     |
    ::: pallets/parachain-staking/src/lib.rs:100:5
     |
100  |     pub struct Pallet<T>(PhantomData<T>);
     |     -------------------- function or associated item `test_benchmark_round_transition_on_initialize` not found for this struct
```
2. It takes too long to generate a weight file via parachain-staking benchmarks code.
```
2022-11-15 14:41:41 Running Benchmark: parachain_staking.join_candidates(1 args) 2/50 1/1
2022-11-15 14:41:50 Running Benchmark: parachain_staking.join_candidates(1 args) 4/50 1/1
2022-11-15 14:41:56 Running Benchmark: parachain_staking.join_candidates(1 args) 5/50 1/1
2022-11-15 14:42:04 Running Benchmark: parachain_staking.join_candidates(1 args) 6/50 1/1
2022-11-15 14:42:14 Running Benchmark: parachain_staking.join_candidates(1 args) 7/50 1/1
2022-11-15 14:42:25 Running Benchmark: parachain_staking.join_candidates(1 args) 8/50 1/1
2022-11-15 14:42:37 Running Benchmark: parachain_staking.join_candidates(1 args) 9/50 1/1
2022-11-15 14:42:51 Running Benchmark: parachain_staking.join_candidates(1 args) 10/50 1/1
2022-11-15 14:43:07 Running Benchmark: parachain_staking.join_candidates(1 args) 11/50 1/1
2022-11-15 14:43:24 Running Benchmark: parachain_staking.join_candidates(1 args) 12/50 1/1
2022-11-15 14:43:43 Running Benchmark: parachain_staking.join_candidates(1 args) 13/50 1/1
2022-11-15 14:44:03 Running Benchmark: parachain_staking.join_candidates(1 args) 14/50 1/1
2022-11-15 14:44:25 Running Benchmark: parachain_staking.join_candidates(1 args) 15/50 1/1
2022-11-15 14:44:54 Running Benchmark: parachain_staking.join_candidates(1 args) 16/50 1/1
2022-11-15 14:45:34 Running Benchmark: parachain_staking.join_candidates(1 args) 17/50 1/1
2022-11-15 14:46:17 Running Benchmark: parachain_staking.join_candidates(1 args) 18/50 1/1
2022-11-15 14:47:10 Running Benchmark: parachain_staking.join_candidates(1 args) 19/50 1/1
2022-11-15 14:48:07 Running Benchmark: parachain_staking.join_candidates(1 args) 20/50 1/1
2022-11-15 14:49:07 Running Benchmark: parachain_staking.join_candidates(1 args) 21/50 1/1
2022-11-15 14:50:10 Running Benchmark: parachain_staking.join_candidates(1 args) 22/50 1/1
2022-11-15 14:51:17 Running Benchmark: parachain_staking.join_candidates(1 args) 23/50 1/1
2022-11-15 14:52:24 Running Benchmark: parachain_staking.join_candidates(1 args) 24/50 1/1
2022-11-15 14:53:41 Running Benchmark: parachain_staking.join_candidates(1 args) 25/50 1/1
2022-11-15 14:54:55 Running Benchmark: parachain_staking.join_candidates(1 args) 26/50 1/1
2022-11-15 14:56:12 Running Benchmark: parachain_staking.join_candidates(1 args) 27/50 1/1
2022-11-15 14:57:27 Running Benchmark: parachain_staking.join_candidates(1 args) 28/50 1/1
2022-11-15 14:58:25 Running Benchmark: parachain_staking.join_candidates(1 args) 29/50 1/1
2022-11-15 14:59:47 Running Benchmark: parachain_staking.join_candidates(1 args) 30/50 1/1
2022-11-15 15:01:17 Running Benchmark: parachain_staking.join_candidates(1 args) 31/50 1/1
2022-11-15 15:02:52 Running Benchmark: parachain_staking.join_candidates(1 args) 32/50 1/1
2022-11-15 15:04:27 Running Benchmark: parachain_staking.join_candidates(1 args) 33/50 1/1
2022-11-15 15:05:29 Running Benchmark: parachain_staking.join_candidates(1 args) 34/50 1/1
2022-11-15 15:06:22 Running Benchmark: parachain_staking.join_candidates(1 args) 35/50 1/1
2022-11-15 15:07:17 Running Benchmark: parachain_staking.join_candidates(1 args) 36/50 1/1
2022-11-15 15:08:13 Running Benchmark: parachain_staking.join_candidates(1 args) 37/50 1/1
2022-11-15 15:09:11 Running Benchmark: parachain_staking.join_candidates(1 args) 38/50 1/1
2022-11-15 15:10:10 Running Benchmark: parachain_staking.join_candidates(1 args) 39/50 1/1
2022-11-15 15:11:11 Running Benchmark: parachain_staking.join_candidates(1 args) 40/50 1/1
2022-11-15 15:12:14 Running Benchmark: parachain_staking.join_candidates(1 args) 41/50 1/1
2022-11-15 15:13:18 Running Benchmark: parachain_staking.join_candidates(1 args) 42/50 1/1
2022-11-15 15:14:24 Running Benchmark: parachain_staking.join_candidates(1 args) 43/50 1/1
2022-11-15 15:15:31 Running Benchmark: parachain_staking.join_candidates(1 args) 44/50 1/1
2022-11-15 15:16:40 Running Benchmark: parachain_staking.join_candidates(1 args) 45/50 1/1
2022-11-15 15:17:50 Running Benchmark: parachain_staking.join_candidates(1 args) 46/50 1/1
2022-11-15 15:19:02 Running Benchmark: parachain_staking.join_candidates(1 args) 47/50 1/1
2022-11-15 15:20:15 Running Benchmark: parachain_staking.join_candidates(1 args) 48/50 1/1
2022-11-15 15:21:30 Running Benchmark: parachain_staking.join_candidates(1 args) 49/50 1/1
2022-11-15 15:22:47 Running Benchmark: parachain_staking.join_candidates(1 args) 50/50 1/1
```
It took too long to execute the benchmark code, I guess it was caused by the low configuration of the computer, which is the configuration of my computer

```
➜  RustCode lscpu
Architecture:                    x86_64
CPU op-mode(s):                  32-bit, 64-bit
Byte Order:                      Little Endian
Address sizes:                   48 bits physical, 48 bits virtual
CPU(s):                          16
On-line CPU(s) list:             0-15
Thread(s) per core:              2
Core(s) per socket:              8
Socket(s):                       1
Vendor ID:                       AuthenticAMD
CPU family:                      25
Model:                           80
Model name:                      AMD Ryzen 9 5900HX with Radeon Graphics
Stepping:                        0
CPU MHz:                         3293.728
BogoMIPS:                        6587.45
Virtualization:                  AMD-V
Hypervisor vendor:               Microsoft
Virtualization type:             full
L1d cache:                       256 KiB
L1i cache:                       256 KiB
L2 cache:                        4 MiB
L3 cache:                        16 MiB
```
```
➜  RustCode free -m
              total        used        free      shared  buff/cache   available
Mem:          32036         176       29626           0        2234       31412
Swap:          8192           0        8192
```

--- a/pallets/parachain-staking/src/benchmarks.rs
+++ b/pallets/parachain-staking/src/benchmarks.rs
@@ -1292,20 +1292,6 @@ mod tests {
 	}
 
 	#[test]
-	fn bench_hotfix_remove_delegation_requests() {
-		new_test_ext().execute_with(|| {
-			assert_ok!(Pallet::<Test>::test_benchmark_hotfix_remove_delegation_requests());
-		});
-	}
-
-	#[test]
-	fn bench_hotfix_update_candidate_pool_value() {
-		new_test_ext().execute_with(|| {
-			assert_ok!(Pallet::<Test>::test_benchmark_hotfix_update_candidate_pool_value());
-		});
-	}
-
-	#[test]
 	fn bench_set_staking_expectations() {
 		new_test_ext().execute_with(|| {
 			assert_ok!(Pallet::<Test>::test_benchmark_set_staking_expectations());
@@ -1498,13 +1484,6 @@ mod tests {
 	fn bench_cancel_delegator_bond_less() {
 		new_test_ext().execute_with(|| {
 			assert_ok!(Pallet::<Test>::test_benchmark_cancel_delegator_bond_less());
-		});
-	}
-
-	#[test]
-	fn bench_round_transition_on_initialize() {
-		new_test_ext().execute_with(|| {
-			assert_ok!(Pallet::<Test>::test_benchmark_round_transition_on_initialize());
 		});
 	}
 

